ddtrace/pin.py

```diff
@@ -56,6 +56,25 @@ def __repr__(self):
         return "Pin(service=%s, app=%s, app_type=%s, tags=%s, tracer=%s)" % (
             self.service, self.app, self.app_type, self.tags, self.tracer)
 
+    @staticmethod
+    def find(*objs):
```

Comment:
Might be worth it to just have a sanity test for `Pin.find`. Don't really think it's necessary to ship this.